storybookjs/react-native
# feat: Move storybook.requires.js and Storybook.tsx files from the root directory to the .storybook folder

Issue: #238 

## What I did
Changed the `loader.js` script (that generates `storybook.requires.js`), the V6 alpha instructions and the example project so that both `storybook.requires.js` and `Storybook.tsx` are moved to the `.storybook` directory from the root.

## How to test
1. Use React Native example project in the `/examples/native` directory. Confirm that the stories still work both on iOS and Android.
2. Run `yarn prepareAll` in the root of the project. Then follow the v6 alpha setup guide in `v6README.md`, but in the same way as in https://github.com/storybookjs/react-native/pull/251 , use the local Storybook React Native dependencies instead of the ones pulled from NPM.

**Note**: In step 2, `cd ios; pod install; cd ..;` fails on my box with the following error:

```
[!] Invalid `Podfile` file: cannot load such file -- /Users/lauriharpf/Repos/lauriharpf/RnSBSixAlpha/node_modules/react-native/scripts/react_native_pods.

 #  from /Users/lauriharpf/Repos/lauriharpf/RnSBSixAlpha/ios/Podfile:1
 #  -------------------------------------------
 >  require_relative '../node_modules/react-native/scripts/react_native_pods'
 #  require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 #  -------------------------------------------
 ```

Running `yarn` in the root of `RnSBSixAlpha` fixes it. I'm unsure why this happens, but I suspect that it might be caused by using the local React Native Storybook dependencies. The same thing happens with the `next-6.0` branch, so it shouldn't be related to this change.

- Does this need a new example in examples/native? **no**, not in my opinion
- Does this need an update to the documentation? **no**, not in my opinion
